netket/models/autoreg.py

```diff
@@ -46,19 +47,43 @@ def __post_init__(self):
             raise ValueError("Only unconstrained Hilbert spaces are supported by ARNN.")
 
     @abc.abstractmethod
-    def conditionals(self, inputs: Array, cache: PyTree) -> Tuple[Array, PyTree]:
+    def _conditional(self, inputs: Array, index: int) -> Array:
         """
-        Computes the probabilities for each spin to take each value.
+        Computes the conditional probabilities for a site to take each value.
```

Comment:
for a site to take each value? 
Shouldn't it be for a variable to take a given value?